cvmfs/cvmfs_config

```diff
@@ -1833,15 +1833,23 @@ EOF
      # SIP) create directory in root `/`.
      # We decide to move the standard installation directory on
      # `/Users/shared/cvmfs` instead of `/cvmfs`
+     # In order to keep the standard CVMFS mountpoint, we decide to implement
+     # firmlinks from `/cvmfs` to `/Users/shared/cvmfs`
      if sw_vers | grep 10.15; then
-        sudo mkdir -p /Users/shared/cvmfs
-        sudo chown cvmfs:cvmfs /Users/shared/cvmfs
-        sudo chmod 777 /Users/shared/cvmfs
+        mkdir -p /Users/shared/cvmfs
+        chown cvmfs:cvmfs /Users/shared/cvmfs
+        chmod 777 /Users/shared/cvmfs
+        if ! grep -q "cvmfs" /etc/synthetic.conf 2> /dev/null ; then
```

Comment:
Perhaps better `grep -q "^cvmfs\t"` to be more precise

Reply:
I intentionally leave it a little unprecise to let user play with it.

With the assumption that they may want to use something different, but that such something will contains `cvmfs` in the name.

I guess is not a big deal to have one more firmlink anyway!

Reply:
It's fine to let the user add more symlinks.  But in this way, if there is some "cvmfs stuff" present in the file, we won't add our version of the symlink.  And I think we should.

Reply:
Fixed!